pkg/detectors/sourcegraph/sourcegraph.go

```diff
@@ -22,7 +22,7 @@ var _ detectors.Detector = (*Scanner)(nil)
 var (
 	defaultClient = common.SaneHttpClient()
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(`\b(sgp_[a-f0-9]{40})\b`)
+	keyPat = regexp.MustCompile(`\b(sgp_(?:[a-fA-F0-9]{16}|local)_[a-fA-F0-9]{40}|sgp_[a-fA-F0-9]{40}|[a-fA-F0-9]{40})\b`)
```

Comment:
This new detector matches the following patterns, right?
```
sgp_0000000000000000_0000000000000000000000000000000000000000
sgp_local_0000000000000000000000000000000000000000
sgp_0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
```

If so, it might be good to split the last one out and add a [prefix](https://github.com/trufflesecurity/trufflehog/blob/a98a4de2fb5c74eeecf8438ff1b7e42b2d3021df/pkg/detectors/tru/tru.go#L25). I don't think it would get flagged right now unless the chunk also happens to include the `sgp_` keyword.